packages/venia-concept/src/components/MiniCart/cartOptions.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component, Suspense } from 'react';
+import { Form } from 'informed';
+import classify from 'src/classify';
+import defaultClasses from './cartOptions.css';
+import Button from 'src/components/Button';
+import Quantity from 'src/components/ProductQuantity';
+import appendOptionsToPayload from 'src/util/appendOptionsToPayload';
+
+const Options = React.lazy(() => import('../ProductOptions'));
+
+class CartOptions extends Component {
+    state = {
+        optionSelections: new Map(),
+        quantity: this.props.cartItem.qty,
```

Comment:
There was a discussion some time ago with the MCE team discouraging component constructors unless necessary for performance reasons, though I don't quite recall the reasoning behind it. That was the motivation behind this approach, so I'm happy to make the change to initializing in `constructor`. Reflected in the latest commit.

Reply:
Oh, thanks.

I was bringing it up for my own education really. [It does look like Babel just puts it in the constructor anyway](https://babeljs.io/repl/#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=MYGwhgzhAEBiD29oFMAeAXZA7AJjASsmMOgHQDC8AtgA7xbbrQDeAUNNBOmJtALwt2HaACMwAJwBc0dAAsAlhFI1x8GkrHihAXwDcrbUA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2017%2Creact%2Cstage-2&prettier=false&targets=&version=6.26.0&envVersion=), which makes sense because you wouldn't have `props` until then.